### PR TITLE
Use FastAPI CLI for local setup

### LINKED ISSUE #462: Use the new FastAPI CLI
https://github.com/tiangolo/fastapi/releases/tag/0.111.0

### PR DESCRIPTION
**Pull request by nitintecg**
### Description

Fixed #462 issue by using FastAPI CLI for local setup 

### Checklist

- [X] Created tests which fail without the change (if possible)
- [X] All tests passing
- [X] Extended the documentation, if necessary

### CODE REVIEW

**Comment by armanddidierjean** [comment]
Hello,

Thank you for your first contribution to Hyperion 🚀

I do have an issue when I try to debug Hyperion with your launch configuration: the server doesn't start and I have an error
```
No module named fastapi.__main__; 'fastapi' is a package and cannot be directly executed
```

Based on [this discussion](https://github.com/tiangolo/fastapi-cli/discussions/20), using `fastapi_cli` instead of `fastapi` in `.vscode/launch.json` solve the issue for me. Do you think this could be a solution?

File: .vscode/launch.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
       "name": "Hyperion",
       "type": "python",
```
Comment (armanddidierjean) [inline .vscode/launch.json:9]: Would you mind changing this to `debugpy` to force the usage of the new Python debugger? The current setting was recently deprecated, and I think this PR would be a great occasion to change it.
https://devblogs.microsoft.com/python/python-in-visual-studio-code-february-2024-release/#python-debugger-extension-installed-by-default

```suggestion
      "type": "debugpy",
```

**Comment by nitintecg** [comment]
> Hello,
> 
> Thank you for your first contribution to Hyperion 🚀
> 
> I do have an issue when I try to debug Hyperion with your launch configuration: the server doesn't start and I have an error
> 
> ```
> No module named fastapi.__main__; 'fastapi' is a package and cannot be directly executed
> ```
> 
> Based on [this discussion](https://github.com/tiangolo/fastapi-cli/discussions/20), using `fastapi_cli` instead of `fastapi` in `.vscode/launch.json` solve the issue for me. Do you think this could be a solution?

yes thanks 👍

**Review by armanddidierjean — APPROVED** [review]
Thank you for your contribution 🎉

**Review by Rotheem — APPROVED** [review]

**Pull request merged**